### ISSUE TITLE
Estimation of the channel by ns-3 and route in DroNS-3

### ISSUE THREAD

**Issue by mvsandoica** [pervasive-computing-systems-group/DroNS3#12]
Hello, 
I would like to know if the estimation of the channel by ns-3 is dependent on the route marked in DroNS-3 and if so, how to get the information of the route followed and the geography of the terrain to optimise the modelling of the channel.

**Comment by mvsandoica** [comment]
Hi @minespecs @JonD07 @etothe2pii 
do you know anything about this issue?
Thanks

**Comment by mvsandoica** [comment]
Hi @JonD07 ,
I am referring to the trajectory followed by the drone.

**Comment by JonD07** [comment]
@mvsandoica No, we are currently taking a snapshot of where the drone is spacially located with regards to the wireless sensor and feeding this information into NS-3 as a static networking scenario. In our application from our _[Holistic Path Planning](https://ieeexplore.ieee.org/abstract/document/10257236)_, we assume that the drone is stationary during communication, which is reflected in DroNS-3.

NS-3 does allow wireless nodes to move, so it should be possible to implement this in an NS-3 script, which could be launched from within DroNS-3. There is an example of this in the CollectData class in commands.py.
